src/ad-ui.js

```diff
@@ -268,7 +268,8 @@ AdUi.prototype.updateAdUi =
   }
   let podCount = ': ';
   if (totalAds > 1) {
-    podCount = ' (' + adPosition + ' of ' + totalAds + '): ';
+    podCount = ' (' + adPosition + ' ' +
+    this.controller.getSettings().adLabelNofN + ' ' + totalAds + '): ';
```

Comment:
Can you indent this line 4 spaces? That falls in line with our style guide. Then we should be good to go. Thanks!